index.html

```diff
@@ -27,40 +27,40 @@
 <head>
 
 	<!-- TO TRANSLATE: "title", "description" -->
-	<title>The Evolution of Trust</title>
-	<meta name="description" content="an interactive guide to the game theory of why &amp; how we trust each other"/>
+	<title>Ewolucja Zaufania</title>
+	<meta name="description" content="Interaktywny przewodnik po zaufaniu według teorii gier"/>
```

Comment:
Mi się nie podoba słowo "mechanizmach", ale jak nic nikomu nie przychodzi lepszego do głowy to spoko, wrzućmy